src/pkg/projects/dir.targets

```diff
@@ -19,6 +19,31 @@
     <StableVersion Condition="'$(IncludePreReleaseLabelInPackageVersion)' != 'true'">$(Version)</StableVersion>
   </PropertyGroup>
 
+  <!--
+    Remove duplicate files returned by PrereleaseResolveNuGetPackageAssets. The resolve task
+    performs extra detection to pick up a PDB file for any file listed in the assets file. This
+    causes duplicates if the assets file lists PDBs. If foo.dll and foo.pdb exist in the package and
```

Comment:
@ericstj, does the problem where `PrereleaseResolveNuGetPackageAssets` gives me duplicate PDB items make sense?

(I believe the duplication happens in other packages too, but package validation is only turned on for WindowsDesktop.)